frontend/src/app/patients/__snapshots__/EditPatient.test.tsx.snap

```diff
@@ -284,6 +284,8 @@ Object {
                         aria-required="true"
                         class="usa-input"
                         id="50"
+                        max="2021-08-17"
```

Comment:
this is going to break every day. Recommend mocking this

Reply:
Addressed in 3afd53bb